etc/config/csharp.amazon.properties

```diff
@@ -2,16 +2,14 @@ compilers=&csharp
 supportsBinary=false
 needsMulti=false
 compilerType=csharp
-defaultCompiler=dotnet601csharp
+defaultCompiler=dotnetMainCs
+
+group.csharp.compilers=dotnetMainCs
+
+compiler.dotnetMainCs.name=.NET R2R (Main)
```

Comment:
can you please restore the original compiler id in the amazon.properties

Reply:
Sure!

Reply:
Reverted all unnecessary changes